ios/FluentUI/Vnext/List/List.swift

```diff
@@ -156,22 +130,36 @@ public struct MSFListView: View {
         }
         return state.sections
     }
+
+    public func overrideTokens(_ tokens: MSFListTokens?) -> MSFListView {
+        state.overrideTokens = tokens
+        return self
+    }
 }
 
 /// Properties that make up section content
-class MSFListSectionStateImpl: NSObject, ObservableObject, Identifiable, MSFListSectionState {
-    var headerTokens: MSFHeaderFooterTokens
-    var id = UUID()
+class MSFListSectionStateImpl: NSObject, ObservableObject, Identifiable, ControlConfiguration, MSFListSectionState {
+    init(style: MSFHeaderFooterStyle = .standard) {
+        let tokens = MSFHeaderFooterTokens()
+        tokens.style = style
```

Comment:
I think you don't need `tokens.style = style` here since we reset `tokens.style` every time tokens or style gets updated.

Reply:
Actually this is necessary, because this line defines the default starting point for `tokens`. It's possible that neither `tokens` nor `style` will be updated from this point on, so having `tokens` start with the correct `style` is crucial.

Reply:
We set both tokens and style in the next three lines.

Reply:
Oh I see what you are saying. So it's important to note that `didSet` is _not_ called during `init()`, since it is written with the assumption that the object in question has been fully initialized (which is not the case during `init`).

Reply:
Ah, gotcha. That makes sense.